README.md

```diff
@@ -1,15 +1,15 @@
 # Create a calorie counter mobile app using Watson Visual Recognition
 
-In this developer journey, we will create a calorie counter mobile app using Apache Cordova, Node.JS and Watson Visual Recognition.This mobile application analyzes the captured image(food) using Watson Visual Recognition and extracts nutritional information.
+In this developer journey, we will create a calorie counter mobile app using Apache Cordova, Node.js and Watson Visual Recognition. This mobile app extracts nutritional information from captured images of food items. 
 
-![Architecture Diagram](https://github.com/RiyaMRoy04/CalorieCounterApp/blob/master/images/arch_diagram.png "Architecture_diagram")
+Currently this mobile app only runs on Android, but can be easily ported to iOS.
+
+![](https://github.com/IBM/CalorieCounterApp/blob/master/images/arch_diagram.png)
```

Comment:
using just `![](images/arch_diagram.png)` would work, also, it should be `doc/architecture.png`